scripts/insert_snapshot_warning.js

```diff
@@ -13,21 +13,30 @@ const WARNING_CSS = fs.readFileSync(path.join(__dirname, 'snapshot_warning.css')
 
 console.log('Inserting snapshot reference warning...');
 
-JSDOM.fromFile('./out/index.html', { contentType: 'text/html; charset=utf-8' }).then((dom) => {
-  const { document } = dom.window;
+const virtualConsole = new VirtualConsole();
+virtualConsole.on('error', () => {
+  // Suppress warnings from e.g. CSS features not supported by JSDOM
+});
 
-  const style = document.createElement('style');
-  style.textContent = WARNING_CSS;
-  document.head.append(style);
+(async () => {
+  let files = ['out/index.html', ...fs.readdirSync('out/multipage').filter(f => f.endsWith('.html')).map(f => 'out/multipage/' + f)];
+  for (let file of files) {
+    console.log(file);
```

Comment:
is this intentional?

Reply:
Yeah it takes a while so it's nice to see progress.